core/src/main/java/org/apache/accumulo/core/conf/ConfigSanityCheck.java

```diff
@@ -98,13 +110,21 @@ else if (!prop.getType().isValidFormat(value))
       log.warn("Use of {} and {} are deprecated. Consider using {} instead.", INSTANCE_DFS_URI, INSTANCE_DFS_DIR, Property.INSTANCE_VOLUMES);
     }
 
-    if (cipherSuite.equals("NullCipher") && !keyAlgorithm.equals("NullCipher")) {
+    if (cipherSuite.equals(NULL_CIPHER) && !keyAlgorithm.equals(NULL_CIPHER)) {
```

Comment:
Would a single XOR-type check suffice to consolidate this and the following check? (with a reworded message, as appropriate):

```java
(cipherSuite.equals(NULL_CIPHER) || keyAlgorithm.equals(NULL_CIPHER)) && !cipherSuite.equals(keyAlgorithm)
```